hm_pyhelper/lock_singleton.py

```diff
@@ -52,16 +52,22 @@ def wrapper_lock_ecc(*args, **kwargs):
                 # try to acquire the ECC resource or may raise an exception
                 lock.acquire(timeout=timeout)
 
-                value = func(*args, **kwargs)
```

Comment:
Could we rename `raise_exception` to `raise_lock_timeout_exception`. To help clarify that other exceptions from `func` will still be raised.